config/default.js

```diff
@@ -142,7 +142,7 @@ module.exports = {
         : 'ws://localhost:8082',
       proposerBaseUrl: process.env.PROPOSER_HOST
         ? `http://${process.env.PROPOSER_HOST}:${process.env.PROPOSER_PORT}`
-        : 'http://172.16.238.1:8100',
```

Comment:
Why did you remove this? How can i access the proposer from outside a container?